tools/class-redirector/class-redirector/ClassRedirectorTask.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+#nullable enable
+
+namespace ClassRedirector
```

Comment:
This is the namespace for our tasks:

```suggestion
namespace Xamarin.MacDev.Tasks
```